### PR TITLE
Add test runner script and a sample fakes3 config file

### PR DESCRIPTION
**Pull request by flei2000**
Please take a look. If you feel it might be helpful for test developers' daily work, please merge it.

### CODE REVIEW

File: ecstest/logger.py [hunk 1]
```diff
@@ -7,7 +7,11 @@ def getLogger():
 
     console_handler = logging.StreamHandler(sys.stdout)
     console_handler.setFormatter(logging.Formatter(formatter))
-    console_handler.setLevel(logging.INFO)
+    log_level = logging.INFO
+    verbose = os.environ.get('ECSTEST_VERBOSE_OUTPUT')
```
Comment (MingatRubicon) [inline ecstest/logger.py]: what shall be the value of this env? looks like it could be anything

File: runtest [hunk 2]
```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
```
Comment (MingatRubicon) [inline runtest]: i know i am sick. i prefer this code in python as well...

File: ecstest/logger.py (hunk 1 above)
Comment (flei2000) [inline ecstest/logger.py]: Exist, or not exist. Actually I'd like to set it to 1 or true if it exists.

Comment (MingatRubicon) [inline ecstest/logger.py]: then had better check if it is 1.

File: runtest (hunk 2 above)
Comment (flei2000) [inline runtest]: It is mostly operating on shell variables. The python code will be strange to set a lot of shell variables, then invoke another shell command.

Comment (flei2000) [inline runtest]: Inconvenient too.

Comment (MingatRubicon) [inline runtest]: ok

File: ecstest/logger.py (hunk 1 above)
Comment (flei2000) [inline ecstest/logger.py]: Set or unset. That's a common usage to use shell var as a switch. Or we need to treat some exceptions (e.g., the var is set to a string), which makes the thing complex.

Comment (MingatRubicon) [inline ecstest/logger.py]: i am ok with that, except that people do not know. we need to have a doc to explain all these env. maybe time to write a man page or add to command online help

File: runtest (hunk 2 above)
Comment (stevendgonzales) [inline runtest]: we can use this for now, until we make way to run "ecstest" which can import nose and call nose.main()

File: ecstest/logger.py (hunk 1 above)
Comment (stevendgonzales) [inline ecstest/logger.py]: This ENV variable should be accessed in our config module, and then used from there

Comment (MingatRubicon) [inline ecstest/logger.py]: agree

**Pull request merged**